src/index.ts

```diff
@@ -66,6 +71,7 @@ export class RetryChunkLoadPlugin {
               error.message = 'Loading chunk ' + chunkId + ' failed after ${
 								this.options.maxRetries
 							} retries.';
+			  ${this.options.lastResortScript ? this.options.lastResortScript : ''}
```

Comment:
if lastResortScript is provided, isn't the expectation is that no error is thrown?

Reply:
@KhodorAmmar I just ported the webpack's version https://github.com/mattlewis92/webpack-retry-chunk-load-plugin/blob/master/src/index.ts#L90-L94C10